src/components/Pocket/TabletPocket.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Image, Text } from 'react-native';
+import PocketIcon from '../../assets/images/PocketIcon.png';
+import Button from '../common/Button';
+import strings from '../../localization/index';
+import styles from './styles';
+
+const TabletPocket = ({ id, time, weight }) => (
+  <View style={styles.containerTabletPocket}>
+    <Image source={PocketIcon} style={styles.pocketImageStyleTablet} />
+    <View style={styles.containerItemTablet}>
+      <Text style={styles.textBlackTablet}>Id:</Text>
+      <Text style={styles.textGrayTablet}>{`#${id}`}</Text>
+    </View>
+    <View style={styles.containerItemTablet}>
+      <Text style={styles.textBlackTablet}>{strings.arrivedAt}</Text>
+      <Text style={styles.textGrayTablet}>{`${time}`}</Text>
+    </View>
+    <View style={styles.containerItemTablet}>
+      <Text style={styles.textBlackTablet}>{strings.weight}</Text>
+      <Text style={styles.textGrayTablet}>{weight === 'Unweighted' ? weight : `${weight} kg`}</Text>
```

Comment:
Use a string from localization strings files instead of 'Unweighted'.

Reply:
Actually, this particular value comes from the APIs possible value for weight which should always be a number (the weight in kg) or the string "Unweighted" when the pocket hasn't been weighted yet.
If we change it to the string, it wouldn't work when the device is set in Spanish.

Reply:
Good call :D